### PR TITLE
Remove '-s' and '--'

### PR DESCRIPTION
**Pull request by vlad-zhukov**
Yarn 1 outputs the following warning:
```
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
```

### CODE REVIEW

**Comment by vlad-zhukov** [comment]
Actually `run`, `-s` and `--` are not needed anymore. In v1 `yarn flow check` will work just fine. It still makes sense to leave `run` just in case.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 language: node_js
 node_js:
-  - '7'
+  - '8'
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH="$HOME/.yarn/bin:$PATH"
```
Comment (jamiebuilds) [inline .travis.yml:6]: Why is this needed?

Comment (vlad-zhukov) [reply .travis.yml:6]: Travis uses yarn@0.27, this is required to update it to the latest. I copied it from [the Jest repo](https://github.com/facebook/jest/blob/5b2b647345a256a842f99f60ebc1eb843448ecc2/.travis.yml).
